internal/k8shandler/collection.go

```diff
@@ -110,7 +110,8 @@ func (clusterRequest *ClusterLoggingRequest) CreateOrUpdateCollection() (err err
 		log.Error(err, "collector.ReconcileServiceMonitor")
 		return err
 	}
-	if err := collector.ReconcilePrometheusRule(clusterRequest.EventRecorder, clusterRequest.Client, collectionSpec.Type, clusterRequest.Forwarder.Namespace, clusterRequest.ResourceNames.CommonName, clusterRequest.ResourceOwner); err != nil {
+
+	if err := collector.ReconcilePrometheusRule(clusterRequest.EventRecorder, clusterRequest.Client, collectionSpec.Type, clusterRequest.Forwarder.Namespace, constants.CollectorName, clusterRequest.ResourceOwner); err != nil {
```

Comment:
@xperimental I wonder if we need to change this log to something like:
* fetch all rules of interest to us
* if CLF > 0, restrict rules to 1
* remove if CLF =0